### PR TITLE
Fix misattributed flake-rust

### PR DESCRIPTION
**Pull request by devinalvaro**

### CODE REVIEW

File: src/pages/start/7.nix-build-flake.mdx [hunk 1]
```diff
@@ -187,7 +187,7 @@ Here's the package definition that enables us to build this Rust package:
 };
 ```
 
-For the full flake, see [`flake.nix`][flake-go] on GitHub or run `cat flake.nix`.
+For the full flake, see [`flake.nix`][flake-rust] on GitHub or run `cat flake.nix`.
```
Comment (cole-h) [inline src/pages/start/7.nix-build-flake.mdx:190]: Thanks for this! I think you'll also need to add a link to the rust flake.nix at the bottom, with the rest of the links: https://github.com/DeterminateSystems/zero-to-nix/pull/192/files#diff-409f0076ad08e4d14c5a022825714a83e8fbfddbaa3bee76287b740a83c5c124R214

**Review by cole-h — CHANGES_REQUESTED** [review]

File: src/pages/start/7.nix-build-flake.mdx (hunk 1 above)
Comment (lucperkins) [reply src/pages/start/7.nix-build-flake.mdx:190]: Actually, it looks like there *is* a URL for the Rust flake but the alias is `flake-rs`. Update this and we're good to go!

Comment (cole-h) [reply src/pages/start/7.nix-build-flake.mdx:190]: Ah, you're right -- I was expecting `flake-rust` and completely skimmed over `flake-rs` :laughing: 

```suggestion
For the full flake, see [`flake.nix`][flake-rs] on GitHub or run `cat flake.nix`.
```

Comment (devinalvaro) [reply src/pages/start/7.nix-build-flake.mdx]: Thanks! @cole-h @lucperkins

**Pull request merged**